lib/aasm/persistence/active_record_persistence.rb

```diff
@@ -189,8 +191,10 @@ def aasm_fire_event(state_machine_name, name, options, *args, &block)
             end
 
             if options[:persist] && success
-              event.fire_callbacks(:after_commit, self, *args)
-              event.fire_global_callbacks(:after_all_commits, self, *args)
+              # Delegating to ActiveRecord
```

Comment:
It looks like this block isn't even meant to be changed at all.  These lines look exactly like the ones this patch deletes, above.